Website/src/items/templates/edgewebsite/cmp entities/Speaker/Data/Role.yml

```diff
@@ -0,0 +1,38 @@
+﻿---
+ID: "213942a2-ceb9-4a63-b34f-25e4f1a688b2"
+Parent: "b3c59735-c022-4854-b407-215c8748ce26"
+Template: "455a3e98-a627-4b40-8035-e683a0331ac7"
+Path: /sitecore/templates/Project/edgewebsite/cmp entities/Speaker/Data/Role
```

Comment:
Why do we still have a "Speaker" CMP entity template? Is "Edge Speaker" replacing this one? If so, please delete this old template.